bin/envconsul-wrapper

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Wraps a command in envconsul, if vault token is present
+
+# Vault init container will drop the token in /vault/token; alternatively we can set the VAULT_TOKEN env variable
+if [ -f /vault/token ]; then
+    vault_token=$(cat /vault/token)
+    export VAULT_TOKEN=$vault_token
+fi
+
+# New versions of vault-init calls the file something slightly different. export vault token if that file is found
+if [ -f /vault/.vault-token ]; then
+    vault_token=$(cat /vault/.vault-token)
+    export VAULT_TOKEN=$vault_token
+fi
+
+function start() {
+    if [ "$1" != "" ]; then
+        cmd="$*"
+        if [ -n "${VAULT_TOKEN}" ]; then
+            echo "have token. starting envconsul"
+            start_envconsul
+        else
```

Comment:
This implies that the command can still run if there is no vault token present. Is this a valid scenario?

Reply:
The command can still run if the env is setup outside of vault. For testing etc.